implementations/rust/ockam/ockam_command/src/tcp/outlet/delete.rs

```diff
@@ -127,9 +134,9 @@ impl DeleteCommandTui for DeleteTui {
         self.terminal()
             .stdout()
             .plain(fmt_ok!(
-                "Outlet with alias {} on Node {} has been deleted",
-                color!(item_name, OckamColor::PrimaryResource),
-                color!(node_name, OckamColor::PrimaryResource)
+                "Outlet with alias {} on node {} has been deleted",
+                color_primary(item_name),
+                color_primary(&node_name)
```

Comment:
Something to keep in mind for a separate PR: we should remove the `color!` macro in favor of the `color_x()` functions.

Reply:
great point; i will make an issue for this